scripts/write_output_tables.R

```diff
@@ -39,7 +39,6 @@ for (current_filter in facets) {
   
   prefix <- paste0(params$platform, "_",
                    params$project_title, "_",
-                   paste(params$current_filter, collapse = "_"), "_",
```

Comment:
This was giving me double underscores... but actually even though they are in subfolders for different facets, they should still retain the $current_filter within the name. Not sure why it isn't getting passed in here via the parameters... Need to troubleshoot that one.